modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comment.es.js

```diff
@@ -27,12 +27,21 @@ import EditedTimestamp from './EditedTimestamp.es';
 import Modal from './Modal.es';
 
 export default withRouter(
-	({comment, commentChange, editable = true, match: {url}}) => {
+	({
+		comment,
+		commentChange,
+		editable = true,
+
+		match: {url},
+	}) => {
 		const context = useContext(AppContext);
 		const [showDeleteCommentModal, setShowDeleteCommentModal] = useState(
 			false
 		);
 		const [deleteMessage] = useMutation(deleteMessageQuery);
+		const companyMxNameLabel = `(${comment.companyMxName
+			?.split('.', 1)
+			.join()})`;
```

Comment:
Simplify

```suggestion
		const [, companyMxName] = comment.companyMxName
			?.split('.');
```

Reply:
With this change in the code, it gave an error and returned the following message "TypeError: (intermediate value)(intermediate value)(intermediate value) is not iterable"